api/paidAction/index.js

```diff
@@ -30,6 +31,7 @@ export const paidActions = {
   TERRITORY_BILLING,
   TERRITORY_UNARCHIVE,
   DONATE,
+  TRANSFER,
```

Comment:
I wonder if `EXTERNAL_P2P` might be a better name? Transfer is just a bit ambiguous in the context of SN.

Reply:
good point, indeed transfer is a bad name

Reply:
on second thought, EXTERNAL_P2P might be as confusing, since this is not always a p2p transfer (if the user has no attached wallets for example). 
Maybe just EXTERNAL or LNURLP ?

Reply:
It can still be considered p2p since they still need to pay any QR codes with own wallets.

However, if this is exclusively about LNURLP, why not call it LNURLP?

(I don't know, I didn't read the rest of the code, I'm just replying to this message)